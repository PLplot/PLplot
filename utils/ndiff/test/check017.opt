--relative-error 1.e-14

