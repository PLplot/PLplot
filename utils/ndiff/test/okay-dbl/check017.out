### Maximum relative error in matching lines = 9.10e-15 at line 38 field 4
